.gitignore

```diff
@@ -14,3 +14,7 @@ pom.xml.releaseBackup
 .idea
 *.sublime-project
 *.sublime-workspace
+
+# vim
+*.swp
+Session.vim
```

Comment:
Run interactive rebase, then reset changes, commit only .gitignore change, commit other changes, continue rebase.
This will split first commit into 2 commits. Probably this what is described here http://stackoverflow.com/a/6217314/4154231